### PR TITLE
webreg: show steps' properties in UI

### PR DESCRIPTION
**Pull request by petr-muller**
Expose additional information about steps in the UI

Test deployment eventually shows up at: https://server-ci-tools-1596.apps.ci.l2s4.p1.openshiftapps.com
Example: https://server-ci-tools-1596.apps.ci.l2s4.p1.openshiftapps.com/reference/gather-loki#properties

### CODE REVIEW

File: pkg/webreg/webreg.go [hunk 1]
```diff
@@ -236,6 +238,79 @@ const templateDefinitions = `
...
+      <td>Step timeout</td>
+      <td>{{ .ActiveDeadlineSeconds }} seconds</td>
+      <td>Limits the execution time of the step.</td>
+    </tr>
+  {{ end }}
+  {{ if .TerminationGracePeriodSeconds }}
+    <tr>
+      <td>Termination grace period</td>
+      <td>{{ .TerminationGracePeriodSeconds }} seconds</td>
+      <td>Period of time until SIGKILL signal is sent to the test pod (after SIGTERM signal is sent).</td>
```
Comment (stevekuznetsov) [inline pkg/webreg/webreg.go:262]: Can we TODO to link to the timeouts page when it's ready?

**Review by stevekuznetsov — APPROVED** [review]
Love it

File: pkg/webreg/webreg.go (hunk 1 above)
Comment (petr-muller) [reply pkg/webreg/webreg.go:262]: Sure!

Comment (petr-muller) [reply pkg/webreg/webreg.go:262]: https://github.com/openshift/ci-tools/pull/1599

**Pull request merged**